librz/analysis/dwarf_process.c

```diff
@@ -869,12 +870,12 @@ static const char *map_dwarf_reg_to_ppc64_reg(ut64 reg_num, VariableLocationKind
 	case 6: return "r6";
 	case 7: return "r7";
 	case 8: return "r8";
-	case 9: return "r8";
-	case 10: return "r9";
-	case 11: return "r10";
-	case 12: return "r11";
-	case 13: return "r12";
-	case 14: return "r13";
+	case 9: return "r9";
```

Comment:
this stuff needs to be rewritten. i don't see any reason to have something like this

Reply:
@wargio good point, added a new commit to simplify this code.